src/RestaurantItem.tsx

```diff
@@ -7,30 +7,27 @@ type RestaurantProps = {
   readonly restaurant: Omit<Restaurant, 'link'>;
   onToggleDrawer: (id?: number) => void;
 };
-class RestaurantItem extends React.Component<RestaurantProps> {
-  render() {
-    return (
-      <li
-        className="restaurant"
-        onClick={() => this.props.onToggleDrawer(this.props.restaurant.id)}
-      >
-        <div className="restaurant__category">
-          <img src={CATEGORY_IMAGES[this.props.restaurant.category]} alt={this.props.restaurant.category} className="category-icon" />
-        </div>
-        <div className="restaurant__info">
-          <h3 className="restaurant__name text-subtitle">
-            {this.props.restaurant.title}
-          </h3>
-          <span className="restaurant__distance text-body">
-            캠퍼스로부터 {this.props.restaurant.distance}분 내
-          </span>
-          <p className="restaurant__description text-body">
-            {this.props.restaurant.description}
-          </p>
-        </div>
-      </li>
-    );
-  }
-}
+const RestaurantItem = (({ restaurant, onToggleDrawer }: RestaurantProps) => {
+  return (
+    <li className="restaurant" onClick={() => onToggleDrawer(restaurant.id)}>
+      <div className="restaurant__category">
```

Comment:
인라인 함수는 가급적 지양해주세요! 상단으로 분리하는 것을 추천드립니다.
props만 잘 정리하면 아래와 같이 적용할 수 있어 보여요.

```suggestion
    <li className="restaurant" onClick={onToggleDrawer}>
      <div className="restaurant__category">
```